### PR TITLE
Switch license to MIT

### LINKED ISSUE #104: Cleanup license type & license holder
Currently project use 3 types licenses

1. MPL 2.0; it's legacy from PSP2SDK, but all files
2. MIT; one file [kernel/rng.h](https://github.com/vitasdk/vita-headers/blob/b3e049f/include/psp2/kernel/rng.h)
3. Not exists; one file [photoexport.h](https://github.com/vitasdk/vita-headers/blob/b3e049f/include/psp2/photoexport.h)

so,

1. @endrift should decide license type of photoexport.h, only she can.
2. @enoposix [said](https://github.com/vitasdk/vita-headers/pull/101#discussion_r89850939) new headers can use MIT.
if we want to change all header to MIT, we need [confirmation](https://github.com/psp2sdk/psp2sdk/commits/all) from @173210 .
3. maybe we need to append `Copyright (C) 2016 vitasdk` or `Copyright (C) 2015-2016 vitasdk` to all headers(but maybe late, already end of year..)

### PR DESCRIPTION
**Pull request by devnoname120**
Close #104

### CODE REVIEW

**Comment by d3m3vilurr** [comment]
can you change `LICENSE.MD` to `LICENSE.md`?

**Comment by d3m3vilurr** [comment]
also if you can, please add license info to doxygen front page, (docs/Mainpage.dox)
we'll lost this information in documents

File: include/psp2/promoterutil.h [hunk 1]
```diff
@@ -62,11 +37,9 @@ int scePromoterUtilityExit(void);
 /**
  * Delete a package from the LiveArea.
  *
- * @param[in] *titleid
- *
  * @return 0 on success.
  */
-int scePromoterUtilityDeletePkg(const char *titleid);
+int scePromoterUtilityDeletePkg(void *unk);
```
Comment (devnoname120) [inline include/psp2/promoterutil.h]: Not sure why this happened.

Comment (devnoname120) [inline include/psp2/promoterutil.h]: I reapplied 948fa6cbe3e002223a02e03981b322fe54fa361f to fix it.

**Pull request merged**